src/App.css

```diff
@@ -1,106 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;700&family=Roboto:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,700;1,100;1,200;1,300&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   list-style: none;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+a {
+  font-family: Roboto Slab, sans-serif;
+  color: #4386bf;
 }
 
 body {
-  background-color: #f1f1f1;
+  background-color: #fafafa;
+  font-family: 'Roboto', sans-serif;
 }
 
-.newbook {
+nav > button {
+  margin-left: auto;
+}
+
+form {
   display: flex;
-  margin: auto;
-  flex-direction: column;
-  justify-content: center;
   align-items: center;
   gap: 1em;
 }
 
+.newbook {
+  border-top: solid 1px #e8e8e8;
+  margin-left: 1%;
+  padding-bottom: 14px;
+  width: 100%;
+  margin-top: 3%;
+}
+
+.newbook > h3 {
+  font-size: 1.25rem;
+  font-family: 'Montserrat', sans-serif;
+  height: 1rem;
+  width: 50%;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍